webnode/src/components/consent-overlay/button-group/index.js

```diff
@@ -1,118 +1,88 @@
 import React from "react";
-import Radium from "radium";
-import color from "color";
+import styled from "styled-components";
 
-class Button extends React.Component {
-  render() {
-    const { onClick, kind, children } = this.props;
-    return (
-      <button onClick={onClick} style={[styles.base, styles[kind]]}>
-        {children}
-      </button>
-    );
+const Button = styled.button`
+  border-radius: 12px;
+  border: none;
+  cursor: pointer;
+  font-family: Poppins;
+  font-size: 23px;
+  font-weight: 600;
+  height: 50px;
+  margin: 15px;
+  min-width: 300px;
+  outline: none;
+  padding: 0;
+  width: 100%;
+  @media (max-width: 1200px) {
```

Comment:
How do we remember to do this on all our components?

Reply:
we could just store it as a config var and then do `${CONFIG.WIDTH}`

Reply:
That would def make it easier. We might just have to be more diligent cause we dont have e2e tests that test layout